addons/website/static/src/snippets/s_process_steps/options.js

```diff
@@ -18,6 +18,16 @@ options.registry.StepsConnector = options.Class.extend({
         this._super(...arguments);
         this.$target.off('.StepsConnector');
     },
+    /**
+     * @override
+     */
+    async updateUI() {
+        if (this.options.editable[0].classList.contains('o_is_mobile')) {
+            this.trigger_up('deactivate_snippet', {$snippet: this.$target});
```

Comment:
What's the point of this?

Reply:
The problem is that the bug I'm fixing here occurs on almost all the options available on the block steps and block columns that are in a steps block when you are in mobile preview.